src/js/bail-trends.js

```diff
@@ -276,38 +277,30 @@ const createCasesScatterPlot = () => {
     ]
   };
 
-  const radiusDesktopMin = 4,
-    radiusDesktop10k = 35;
-  const radiusMobileMin = 4,
-    radiusMobile10k = 21;
+  const _100_cases_px_desktop = 4; // 100 total cash bail cases === 4px radius circle for desktop screens
+  const _25K_cases_px_desktop = 35; // 25,000 total cash bail cases === 40px radius circle for desktop screens
+  const _100_cases_px_mobile = 4; // 100 total cash bail cases === 4px radius circle for mobile screens
+  const _25K_cases_px_mobile = 21; // 25,000 total cash bail cases === 20px radius circle for mobile screens
 
-  const maxRadiusValue = Math.max(
-    ...Object.values(BAIL_CASES_SCATTER_PLOT).map((county) => county.r)
-  );
-  const minRadiusValue = 100;
+  /* The scaleSqrt scale is useful for sizing circles by area (rather than radius).
+     (When using circle size to represent data, it’s considered better practice to set the area,
+     rather than the radius proportionally to the data.)
+     https://eagereyes.org/blog/2008/linear-vs-quadratic-change
```

Comment:
Ah good thinking; I definitely think this is the best approach. Agreed we'll have to confirm with Rong but I think this is right.